modules.conf

```diff
@@ -5,7 +5,7 @@
 # <module> <min_version> [ <max_version> ]
 
 AnyEvent 5.202
-Audio::Scan 0.93 0.99
+Audio::Scan 0.93 1.99
```

Comment:
Let's remove this. We could still add it back once a version was known to be broken.